manifest_staging/charts/gatekeeper/templates/namespace-post-install.yaml

```diff
@@ -88,6 +88,9 @@ rules:
       {{- range .Values.postInstall.labelNamespace.extraNamespaces }}
       - {{ . }}
       {{- end }}
+{{- with .Values.postInstall.labelNamespace.extraRules }}
```

Comment:
looks like @ritazh commented in https://github.com/open-policy-agent/gatekeeper/pull/2064/files#r882236998 to be consistent with similar implementations

@ritazh do you mean handling this in helmify code?

Reply:
@sozercan 
The issue with using helmify here lies in how the kustomization works
some of the manifests are used in helmify and have a substring to be replaced with helm

but the role specified for the manager is https://github.com/open-policy-agent/gatekeeper/blob/master/config/rbac/role.yaml
so you can't really use helmify and have to use the main.go because it is static but goes through the kustomization process 

the templates i have changed are also static and do not go through a process of helmifiy, so I cant us the helmify or the main.go
you just do copy directories  - https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/main.go#L225